client/packages/system/src/Name/ListView/ImportProperties/ImportReviewDataTable.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { FC, useState } from 'react';
+import {
+  ColumnDescription,
+  DataTable,
+  Grid,
+  NamePropertyNode,
+  NothingHere,
+  Pagination,
+  PropertyNode,
+  SearchBar,
+  TooltipTextCell,
+  useColumns,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { ImportRow } from './PropertiesImportModal';
+
+interface ImportReviewDataTableProps {
+  rows: ImportRow[];
+  properties: NamePropertyNode[] | undefined;
+}
+export const ImportReviewDataTable: FC<ImportReviewDataTableProps> = ({
+  rows,
+  properties,
+}) => {
+  const t = useTranslation();
+  const [pagination, setPagination] = useState<Pagination>({
+    page: 0,
+    first: 20,
+    offset: 0,
+  });
+  // Could filter here for only properties that are used in import
+  const propertyNodes: PropertyNode[] | undefined = properties
+    ?.map(property => {
+      return { ...property.property };
+    })
+    .sort();
+
+  const [searchString, setSearchString] = useState<string>(() => '');
+  const columnDescriptions: ColumnDescription<ImportRow>[] = [
+    {
+      key: 'code',
+      width: 50,
+      sortable: false,
+      label: 'label.code',
+    },
+    {
+      key: 'name',
+      width: 150,
+      sortable: false,
+      label: 'label.name',
+      Cell: TooltipTextCell,
+    },
+  ];
+  propertyNodes?.map(property =>
+    columnDescriptions.push({
+      key: property.key,
+      width: 100,
+      sortable: false,
+      label: undefined,
+      labelProps: { defaultValue: property.name },
+      Cell: TooltipTextCell,
+    })
+  );
+
+  {
+    columnDescriptions.push({
+      key: 'errorMessage',
+      label: 'label.error-message',
+      width: 150,
+      Cell: TooltipTextCell,
+    });
+  }
+
+  const rowsWithProperties = rows.map(row => {
```

Comment:
Add properties to review data table.

We don't have this in other custom properties CSV import - but I figured this makes sense because this import is specifically for properties import. Also in this case we upsert changes, so helpful to see what is being removed as well as added.

However I'm not sure how it will look with many custom properties where it could extend far off the page

Reply:
Really like being able to review the properties! I reckon while we have a defined set of properties, this is great... perhaps if we get to the time where there are loads of them, we can do as you commented above and filter columns to only those that have changed? But I don't think scrolling to review is the end of the world in this case 🤷‍♀️